addons/mail/static/src/discuss/call/common/call.js

```diff
@@ -43,8 +43,10 @@ export class Call extends Component {
         this.grid = useRef("grid");
         this.notification = useService("notification");
         this.rtc = useState(useService("discuss.rtc"));
+        this.ui = useService("ui");
         this.state = useState({
             isFullscreen: false,
+            isCompactCall: false,
```

Comment:
I would avoid name "compact" because this is already reserved with the props `compact`.

Let's use `collapsed` for now.

We should clean the code, because `compact`/`minimized` are quite synonymous and unclear with new `collapsed` feature. I think we should just use `env` modifiers on chatWindow rather than the passing of a dedicated prop.

The cleanup is for another PR ;)